{{cookiecutter.project_dashed}}-src/Pipfile

```diff
@@ -20,7 +21,7 @@ flask-webtest = "*"
 mock = "*"
 "psycopg2-binary" = "*"
 pyquery = "*"
-pytest = "*"
+pytest = "<4.1"
```

Comment:
Leave a comment regarding need for the pinned dependency and link to one of our issues for more detail/resolution status?